logging.go

```diff
@@ -76,9 +76,6 @@ func (a *adapter) logit(msg string, keyvals []interface{}, iserror bool) {
 		keyvals = append(keyvals, ErrMissingLogValue)
 	}
 	m := (len(a.keyvals) + 1) / 2
-	if len(a.keyvals)%2 != 0 {
-		a.keyvals = append(a.keyvals, ErrMissingLogValue)
```

Comment:
I couldn't think of a test to add -- this line is basically unreachable code, as far as I can tell.

`a.keyvals` will always have an even length, thanks to the check in New(): https://github.com/jim-slattery-rs/goa/blob/72a07361f9b5491636eeacaae3b3145b6fe63e96/logging.go#L62-L64

Therefore, we fortunately don't have to check it again here, since it would anyways be unsafe to mutate `a.keyvals`.